src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -9,6 +18,14 @@ export default (accessToken, userID, opts) => {
   )
 
   return {
+    startIdentityVerificationLoader: ({
+      identityVerificationId,
+    }: StartIdentityVerificationGravityInput) => {
+      return Promise.resolve<StartIdentityVerificationGravityOutput>({
+        identity_verification_id: identityVerificationId,
+        identity_verification_wizard_url: "https://staging.artsy.net/auctions",
```

Comment:
just wanted any old URL-like-string here. Will be replaced by return value from Gravity.

Reply:
any reason to not have the return value just be a standard (updated) Identity Verification object?

Reply:
Yeah good question. I was thinking about a couple of things:

1. [minor] I wanted to do what was minimal to get this mutation exposed into MP, acknowledging that it seems likely that we'll want to iterate on the shape and structure here.

2. [major] the wizard URL isn't really part of an identity verification in the sense that we aren't going to persist or consistently expose the wizard URL. It's very much a one-time bit of data. As such, it seems not `part of` an identity verification. So instead of modeling out an `IdentityVerification` in MP in this PR, I think it makes the most sense to defer that modeling into we're forced to implement query-side code to support the Reaction interfaces.